.github/ISSUE_TEMPLATE/1-bug.yml

```diff
@@ -31,7 +31,7 @@ body:
     validations:
       required: true
 
-  - type: textarea
+  - type: input
```

Comment:
I intentionally set this to `textarea`.

Yes, it's just an input line. However, input lines are always rendered by GitHub as plain text, and for text areas you can force it to put fences (```) around them:

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#input

If I kept it as an input line, it seemed to me that it will be too easy for users to destroy the output formatting.

Reply:
did you try that?
and how are input different from textareas? i fail to see the difference.

Reply:
Yes, of course I tried that. Did you?

https://github.com/zyv/test-github-issue-templates/issues/1

Notice how the value of the input is inserted literally and the value of the textarea is enclosed as a block of code.

Reply:
no, because i don't know how. just enabling issues in my clone didn't help.

but i tried messing with your test repo, and as you can see, GH clearly escapes/filters my attempts. so i think an input for the os will be ok.

Reply:
well, a rickroll is silly, but not really bad.

anyway, you can see in issue 4 that one can mess up things up even with textfields. so this is actually a lost cause. and i mean ... big deal. if somebody tries hard to get themselves banned, let them.